tests/test_pybats.py

```diff
@@ -1070,6 +1070,10 @@ def setUp(self):
                                        'log_ram_test2.log')
         self.ionopot_nc3 = os.path.join(spacepy_testing.datadir, 'pybats_test',
                                         'ram_iono_pot.nc')
+        self.bplas_test = os.path.join(spacepy_testing.datadir, 'pybats_test',
+                                       'bound_plasma_t000030.out')
+        self.bpgroup_test = os.path.join(spacepy_testing.datadir,
+                                         'pybats_test/')
```

Comment:
If we need the trailing slash, do `'pybats_test`, '')` rather than the explicit slash.